nengo/tests/test_connection.py

```diff
@@ -611,6 +612,23 @@ def test_set_eval_points(Simulator):
     Simulator(model)  # Builds fine
 
 
+@pytest.mark.parametrize('eval_points, radius', [
+    (UniformHypersphere().sample(500, 1), 0.5),
+    (UniformHypersphere().sample(500, 1), 1.),
+    (UniformHypersphere().sample(500, 1), 1.5)])
+def test_eval_points_scaling(Simulator, eval_points, radius, seed):
+    model = nengo.Network(seed=seed)
+    with model:
+        a = nengo.Ensemble(1, 1, radius=radius)
+        b = nengo.Ensemble(1, 1)
+        con = nengo.Connection(a, b, eval_points=eval_points)
+
+    sim = Simulator(model)
+    dists = np.linalg.norm(sim.data[con].eval_points, axis=1)
+    assert np.all(dists <= radius)
+    assert np.any(dists >= 0.9 * radius)
```

Comment:
I'm not sure if we need this test, since it's the same as for Ensemble in #595. I realize that you put it in because there was a difference between Ensemble and Connection that shouldn't have been there, but sometimes I feel like we get into the habit of finding a bug, fixing the bug, then adding a test that tests whether that specific bug is fixed. This means we end up with a lot of tests, and I'm not really sure how useful it is since there's no way we could test every possible bug, and we really want our tests to highlight bugs for us, not the other way around.

But all that is a general comment on how we do tests sometimes; sorry to take it all out on this test!